src/Dfe.PlanTech.Application/Content/Queries/GetCategorySectionsQuery.cs

```diff
@@ -6,16 +6,10 @@
 
 namespace Dfe.PlanTech.Application.Content.Queries;
 
-public class GetCategorySectionsQuery : IGetPageChildrenQuery
+public class GetCategorySectionsQuery(ICmsDbContext db, ILogger<GetCategorySectionsQuery> logger) : IGetPageChildrenQuery
 {
-    private readonly ICmsDbContext _db;
-    private readonly ILogger<GetCategorySectionsQuery> _logger;
-
-    public GetCategorySectionsQuery(ICmsDbContext db, ILogger<GetCategorySectionsQuery> logger)
-    {
-        _db = db;
-        _logger = logger;
-    }
+    private readonly ICmsDbContext _db = db;
+    private readonly ILogger<GetCategorySectionsQuery> _logger = logger;
```

Comment:
No real need to change but with this change can you just swap all uses of `_db` for `db` and drop these readonly vars? fine as is just a question from me really as to if theres any gain from them

Reply:
Yup. And I know you said there's no need to change but I've done it to simplify the code.